### PR TITLE
Updated the "html" databricks-geocoding notebook with "scala" notebook (branch release-5.0)

### PR DESCRIPTION
**Pull request by rajveer-singh-01**
Updated the "html" databricks-geocoding notebook with "scala" notebook

### CODE REVIEW

**Review by KBouquot — APPROVED** [review]

File: databricks-geococoding/Geocoding_Installation_SDM_To_DBFS.scala [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+// COMMAND ----------
+
+// MAGIC %md ## Install the SDK
+
+// COMMAND ----------
+
+// DBTITLE 1,Install SDK to DBFS
+// MAGIC %sh . /dbricks_env.sh
+// MAGIC 
+// MAGIC mkdir -p DBFS_SDK_EXTRACT_LOCATION
```
Comment (KBouquot) [inline databricks-geococoding/Geocoding_Installation_SDM_To_DBFS.scala]: missing $ on the var name

Comment (rajveer-singh-01) [reply databricks-geococoding/Geocoding_Installation_SDM_To_DBFS.scala]: Will correct it.

**Review by KBouquot — APPROVED** [review]

**Review by KBouquot — APPROVED** [review]

**Comment by rnavarette** [comment]
the scala files look reasonable on first pass... couple questions though
1) before merge, make sure someone has verified them not just through review but after import
2) do these files have any changes related to 5.0?  I thought that this change would be in 4.1 and dev, and then 5.0 branch would be created from dev.  Or, is this just a case of we created 5.0 branch before this feature was in place?  This is just a book-keeping concern i think and not a big deal as long as no 5.0 packaging/deployment stuff goes into 4.1

**Comment by rajveer-singh-01** [comment]
Updated the databricks-geocoding notebooks to scala notebook and updated the readme for v5.0 related changes.

**Pull request merged**